python_modules/dagster/dagster/_core/host_representation/repository_location.py

```diff
@@ -449,7 +449,7 @@ def get_external_schedule_execution_data(
         instance: DagsterInstance,
         repository_handle: RepositoryHandle,
         schedule_name: str,
-        scheduled_execution_time,
+        scheduled_execution_time: datetime.datetime,  # actually a pendulum datetime
```

Comment:
isn't it better to leave this untyped? why do this if you have the type: ignore below?